docs/InstallGuide.md

```diff
@@ -1,424 +1,366 @@
+Create 2 LXC guests
+(it may require rsync to be installed)
 
-+ vm1, vm3 -> vnet1
-+ vm2, vm4 -> vnet2
+```
+# lxc-create -t centos -n inst1
+# lxc-create -t centos -n inst2
+```
 
-The command "run-vm-all" starts all four virtual machines at once.
+Configure interfaces of each guest
 
-    # cd /root/vnet-test-kvm/vm
-    # ./run-vm-all
-    *** vm1 ***
-    node-id: 1
-    interface name: if-x5vjmksl
-    mac address: 52:54:00:0d:84:8c
-    *** vm2 ***
-    node-id: 2
-    interface name: if-sve1s6xp
-    mac address: 52:54:00:9a:cc:00
-    *** vm3 ***
-    node-id: 3
-    interface name: if-bvixz40n
-    mac address: 52:54:00:0a:fb:78
-    *** vm4 ***
-    node-id: 4
-    interface name: if-7st2alal
-    mac address: 52:54:00:e5:f9:fb
+```
+# vi /var/lib/lxc/inst1/config
+lxc.network.type = veth
+lxc.network.flags = up
+lxc.network.veth.pair = inst1
+lxc.network.ipv4 = 10.100.0.10
+lxc.network.hwaddr = 10:54:FF:00:00:01
+lxc.rootfs = /var/lib/lxc/inst1/rootfs
+lxc.include = /usr/share/lxc/config/centos.common.conf
+lxc.arch = x86_64
+lxc.utsname = inst1
+lxc.autodev = 0
+```
 
-### Check the IP address of virtual machines
+```
+# vi /var/lib/lxc/inst2/config
+lxc.network.type = veth
+lxc.network.flags = up
+lxc.network.veth.pair = inst2
+lxc.network.ipv4 = 10.100.0.11
+lxc.network.hwaddr = 10:54:FF:00:00:02
+lxc.rootfs = /var/lib/lxc/inst2/rootfs
+lxc.include = /usr/share/lxc/config/centos.common.conf
+lxc.arch = x86_64
+lxc.utsname = inst2
+lxc.autodev = 0
+```
 
-You can log into the virtual machines using the telnet command.
+We do not use `lxc.network.link` parameter because the linux bridge is replaced by the Open vSwitch.
 
-    # telnet localhost PORT-NO
+Make sure that the IPv4 address and MAC address are the same as what you specify when you create the interface database records.
 
-The port numbers are:
+Launch the LXC guests then enslave the LXC's tap interfaces to the datapath.
 
-+ vm1 : 5001
-+ vm2 : 5002
-+ vm3 : 5003
-+ vm4 : 5004
+```
+# lxc-start -d -n inst1
+# lxc-start -d -n inst2
 
-The ip addresses are assigned as follows. As you can see, vm1 and vm3, share the same ip address, as do vm2 and vm4.
+# ovs-vsctl add-port br0 inst1
+# ovs-vsctl add-port br0 inst2
+```
 
-+ vm1 : 10.102.0.10
-+ vm2 : 10.102.0.10
-+ vm3 : 10.102.0.11
-+ vm4 : 10.102.0.11
+Now the LXC's network interfaces are attached to the Open vSwitch, likewise you plug a LAN cable to a network switch.
 
-Log into the virtual machines and make sure their ip addresses are correct.
+Log in to the inst1 to see if the IP address is assigned properly.
 
-**ex) vm1**
+```
+# lxc-console -n inst1
+# ip a
+```
 
-    # telnet localhost 5001 <- vm1
-    Trying 127.0.0.1...
-    Connected to localhost.
-    Escape character is '^]'.
+The IP address is assigned to eth0. After the network interface of inst1 is attached to br0, a DHCP request is sent then the IP address is determined by the DHCP server implemented in vna.
```

Comment:
I know I suggested this DHCP line before, but that was because I mistakenly assumed the we would be using OpenVNet's simulated DHCP service. Since we're not using it, this line should be removed.